frontend/test/metabase/lib/expressions/typechecker.unit.spec.js

```diff
@@ -110,6 +110,10 @@ describe("type-checker", () => {
     it("should accept a CASE expression with complex arguments", () => {
       expect(() => validate("CASE(Deal, 0.5*X, Y-Z)")).not.toThrow();
     });
+
+    it("should accept PERCENTILE with two arguments", () => {
+      expect(() => validate("PERCENTILE([Rating], .5)")).not.toThrow();
```

Comment:
I have to write `0.5` - it fails with `.5` - not sure how the unit test handles this - I'm on Firefox
![image](https://user-images.githubusercontent.com/1447303/115616271-4d707b00-a2f0-11eb-80ed-b0e94b8ff9a9.png)

Reply:
Great catch @flamber! Seems that the "Unknown Field" error came from the MBQL compiler. The unit tests skip that, it goes straight to the type checker (to verify the correct function arguments).
I'll investigate that as a separate issue.